pkg/rancher-desktop/main/commandServer/httpCommandServer.ts

```diff
@@ -537,6 +538,36 @@ export class HttpCommandServer {
     }
   }
 
+  async k8sReset(request: express.Request, response: express.Response, context: commandContext): Promise<void> {
+    let values: Record<string, any> = {};
+    const [data, payloadError] = await serverHelper.getRequestBody(request, MAX_REQUEST_BODY_LENGTH);
+    let error = '';
+    let mode: 'fast' | 'wipe' = 'fast';
+
+    if (!payloadError) {
+      try {
+        console.debug(`Request data: ${ data }`);
+        values = JSON.parse(data);
+        if ('mode' in values && typeof values.mode === 'string' && (values.mode === 'fast' || values.mode === 'wipe')) {
+          mode = values.mode;
+        }
```

Comment:
Should this return an error when `mode` is set to an invalid value? Right now it effectively recognizes `wipe`, and matches every other value, including missing values, to `fast`.